sdk/keyvault/azkeys/client.go

```diff
@@ -237,26 +240,28 @@ func (c *Client) CreateECKey(ctx context.Context, name string, options *CreateEC
 	return createECKeyResponseFromGenerated(resp), nil
 }
 
-// CreateOctKeyOptions contains the optional parameters for the Client.CreateOCTKey method
+// CreateOctKeyOptions contains optional parameters for CreateOctKey
```

Comment:
We can handle separately, but I think that `Oct` should be capitalized.

Reply:
Sure, let's discuss it elsewhere. Counterargument is that "oct" is short for "octet sequence"; it isn't an acronym like "EC" and "RSA".

Reply:
Good point, no need for further discussion.